### DISCUSSION TITLE
Pyrasite

### DISCUSSION THREAD

**Discussion by wormstein** [EdNekebno/DGTCentaurMods#4] [Q&A] [answered]
Hi,
I saw your video on Twitter. I was wondering where are you running pyrasite? Is it on the original software that comes with the board?

I am trying to port everything on a RPi 0 W and make it work as smooth as possible and apparently pyrasite don't work on Raspbian OS... Might be Yama but not sure. It just hangs.

-Adrian

**Comment by EdNekebno** [comment, marked as answer]
I installed it. I'm pretty sure I made the changes described here: https://github.com/lmacken/pyrasite/commit/d1e2301fe60ab03e99a51ca1b105f7c3368b1a33 to make it work on raspbian.

**Reply by wormstein** [reply]
Hah, it works.

I went back and reviewed all injector.py to make sure this change is on all of them. There were two on the system...

Thanks
